src/main/java/build/buildfarm/instance/stub/Chunker.java

```diff
@@ -175,6 +175,7 @@ public Chunk next() throws IOException {
     maybeInitialize();
 
     if (size == 0) {
+      data.close();
```

Comment:
This should just call `reset()` and avoid the cleanup behaviors. It should be the only location that a close occurs, and it includes (and can function properly) regardless of the state of `data`

Reply:
Fixed